components/clubhouse_io/clubhouse.app.js

```diff
@@ -0,0 +1,156 @@
+const clubhouse = require("clubhouse-lib");
+const get = require("lodash/get");
+const retry = require("async-retry");
+
+module.exports = {
+  type: "app",
+  app: "clubhouse",
+  methods: {
+    _authToken() {
+      return this.$auth.api_key;
+    },
+    _clubhouseio() {
```

Comment:
i personally prefer to name the method accesing the sdk to the application or services being used, in this case clubhouse, but that just me. i didn't see api() in airtable, but base(). api is in mailgun thought, i'll be changing to api() but i wonder if this will be standarized across all components.

Reply:
ok this was changed.